package.json

```diff
@@ -22,8 +24,12 @@
     "vuex": "^3.0.1"
   },
   "devDependencies": {
+    "@storybook/addon-actions": "^6.0.22",
+    "@storybook/addon-essentials": "^6.0.22",
+    "@storybook/addon-links": "^6.0.22",
+    "@storybook/vue": "^6.0.22",
     "@types/jest": "^23.1.4",
-    "@vue/cli-plugin-babel": "^3.12.0",
+    "@vue/cli-plugin-babel": "^4.5.6",
```

Comment:
Any idea why we wouldn't get the latest major version when I initialized the project?

Reply:
The Vue create scripts still use v3, which is why it was created that way. Since the dependency version is prepended with the ^, it means we'll pull the latest minor/patch versions of the major version 3.

Vue has a guide for migrating to v4 here: https://cli.vuejs.org/migrating-from-v3/#migrating-from-v3